services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/tormaystarkastelu/TormaystarkasteluTulos.kt

```diff
@@ -0,0 +1,22 @@
+package fi.hel.haitaton.hanke.tormaystarkastelu
+
+class TormaystarkasteluTulos {
+    var hankeId : Int = 0
+    var liikennehaittaIndeksi: LiikennehaittaIndeksiType? = null
```

Comment:
Do we need to have this calculated result here at all? Would it be the same if we had only the three different values and the client (UI) would see from they which is the biggest and what type it is?

So, instead of this:
```
{
  "hankeId" : 1,
  "liikennehaittaIndeksi" : {
    "indeksi" : 2.3,
    "type" : "JOUKKOLIIKENNEINDEKSI"
  },
  "perusIndeksi" : 2.2,
  "pyorailyIndeksi" : 1.7,
  "joukkoliikenneIndeksi" : 2.3
}
```
there would be this:
```
{
  "hankeId" : 1,
  "perusIndeksi" : 2.2,
  "pyorailyIndeksi" : 1.7,
  "joukkoliikenneIndeksi" : 2.3
}
```
Plus perhaps `hankeTunnus` also.

Reply:
I think that leaves too much decisions for ui? We also need the one number for Hanke to decide the color quickly in map.

Reply:
The decision for color would still be a trivial calculation, that is not a good reason. However, I also think the main index to be calculated in the backend; if backend does some calculations/decisions, it should do them all, and not split this logic between backend and fronted. (Also, I expect that the main index calculation will get more complex over time..)